ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -752,3 +752,5 @@ job_request: "{{ env_name }}_job_request"
 
 #### Exhuats API consumer 
 exhaust_api_consumer_ids: ["273f3b18-5dda-4a27-984a-060c7cd398d3"] # being used in analytics api to access the analytics exhaust api
+user_profile_read_url: "https://{{ domain_name }}/api/user/v2/read/" # being used in analytics api to get user details
+org_search_url: "https://{{ domain_name }}/api/org/v1/search" # being used in analytics api to get mhrd tenant id
```

Comment:
call should be done via internal service

Reply:
For this change will give new PR